cmd/gator/test/test.go

```diff
@@ -107,8 +115,23 @@ func runSuites(ctx context.Context, fileSystem fs.FS, suites []gktest.Suite, fil
 				}
 			}
 		}
+
+		results[i] = suiteResult
+		i++
+
+		w := &strings.Builder{}
+		printer := gktest.PrinterGo{}
+		err := printer.Print(w, results, verbose)
+		if err != nil {
+			return err
+		}
+		fmt.Println(w)
 	}
 
+	sort.Slice(results, func(i, j int) bool {
```

Comment:
Are we using `results` after it's sorted?

Reply:
Nope. Removed.